src/controllers/skills/skillsController.ts

```diff
@@ -1,6 +1,7 @@
 import {NextFunction, Request, Response, Router} from 'express'
-import {CsrsService} from '../csrs/service/csrsService'
+import {CsrsService} from '../../csrs/service/csrsService'
 import * as csvtojson from 'csvtojson'
+import {PlaceholderDateSkills} from '../../learning-catalogue/model/placeholderDateSkills'
 
 class Choice {
```

Comment:
We decided this object will be called Answer in the design meeting. Can you rename this one pl